modules/uhsg_chat/config/install/uhsg_chat.config.yml

```diff
@@ -0,0 +1,2 @@
+# Display the chat on the following nodes (identified by nid):
+display_chat_on_nodes: []
```

Comment:
As we are introducing configuration to Drupal, we should provide schema information too. [Read more](https://www.drupal.org/docs/8/api/configuration-api/configuration-schemametadata#use) about why to use schema declarations.

Based on `[]` default value, it looks like it's a type of `sequence`.